src/common/card/card.module.scss

```diff
@@ -0,0 +1,63 @@
+@import 'spacings';
+@import 'fonts';
+
+* {
```

Comment:
This `*` would probably have unwanted side effects on many elements. Better to stick to `classNames`.

Reply:
Right, I have removed this.